Gemfile

```diff
@@ -1,3 +1,5 @@
 source "https://rubygems.org"
 
+gem 'rspec', :require => 'spec'
```

Comment:
I don't really know anything about ruby packaging, so I'm just curious, why add this to the Gemfile when it's already in `hocon.gemspec`?